react/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "vtex",
+  "env": {
+    "es6": true,
+    "jest": true,
+    "node": true
+  },
+  "rules": {
+    "prettier/prettier": "off",
```

Comment:
where do these eslint rules come from?

have you followed this recipe? https://docs.google.com/document/d/1ePi5Mn3pe1dOorKrMtC-MclBDiIDimVzf0Uj61W8lA4

I think it's a good standard to follow